src/views/transactions/exit/exit.view.tsx

```diff
@@ -0,0 +1,285 @@
+import React from "react";
+import { connect } from "react-redux";
+import { useLocation } from "react-router";
+import { BigNumber } from "@ethersproject/bignumber";
+import { push } from "connected-react-router";
+
+import {
+  Account,
+  FiatExchangeRates,
+  HermezWallet,
+  PooledTransaction,
+  RecommendedFee,
+  Token,
+} from "src/domain/hermez";
+import { AppDispatch, AppState } from "src/store";
+import { AsyncTask } from "src/utils/types";
+import useExitStyles from "src/views/transactions/exit/exit.styles";
+import * as exitActions from "src/store/transactions/exit/exit.actions";
+import * as exitThunks from "src/store/transactions/exit/exit.thunks";
+import { EstimatedWithdrawFee, Header } from "src/domain";
+import { changeHeader } from "src/store/global/global.actions";
+import Spinner from "src/views/shared/spinner/spinner.view";
+import TransactionForm from "src/views/transactions/components/transaction-form/transaction-form.view";
+import TransactionOverview from "src/views/transactions/components/transaction-overview/transaction-overview.view";
+import { TxType } from "@hermeznetwork/hermezjs/src/enums";
+
+interface ExitState {
```

Comment:
In the `transfer.view.tsx` I did name these interfaces `TransferViewState`, `TransferViewHandlers` and `TransferViewProps`.

For the sake of consistency (and for me to sleep peacefully), can you either add the suffix "View" here or remove it there?